test/unit/pages/location/controller-test.js

```diff
@@ -30,4 +50,40 @@ describe('LocationController', function() {
     });
 
   });
+
+  describe('setCloseDialog', function() {
+
+    it('should able to showCloseDialog', function() {
+      locationcontroller.setCloseDialog();
+      expect(locationcontroller.showCloseDialog).to.be.equal(true);
+    });
+  });
+
+  describe('checkUserProfileLoaction', function() {
```

Comment:
checkUserProfileLoaction  function is  vagrant related validation. Since vagrant is going to be used for CDK 3 release, lets skip any test improvements related to vagrant.